slack.md

```diff
@@ -28,10 +28,10 @@ available for help.
 
 ## Channels
 
-* [#general](https://{{site.slack_fqdn}}/messages/CFLB9AMBN) - discussions about using and developing Trino
+* [#announcements](https://{{site.slack_fqdn}}/messages/CFLB9AMBN) - announcements about Trino
```

Comment:
I thought #general is going to be used for announcements?

Reply:
seems like we renamed .. so is the default channel that everyone automatically now joins #announcements ?

Reply:
Yes